### PR TITLE
Allow to not modify `ProcessManager` state

### PR DESCRIPTION
**Pull request by dmytro-grankin**
This PR resolves the [issue](https://github.com/SpineEventEngine/core-java/issues/585). 

Also small issues in the related Javadocs and tests was fixed.

### CODE REVIEW

**Comment by dmytro-grankin** [comment]
@armiol PTAL.

File: server/src/main/java/io/spine/server/procman/PmEventEndpoint.java [hunk 1]
```diff
@@ -71,9 +71,13 @@ private PmEventEndpoint(ProcessManagerRepository<I, P, ?> repository, EventEnvel
         return events;
     }
 
+    /**
+     * Does nothing since a state of a process manager should not be necessarily
+     * updated during reacting on an event.
```
Comment (armiol) [inline server/src/main/java/io/spine/server/procman/PmEventEndpoint.java]: `upon reacting`

File: server/src/main/java/io/spine/server/projection/ProjectionEndpoint.java [hunk 2]
```diff
@@ -110,9 +110,13 @@ protected void onModified(P projection) {
                                     .getTenantId(), projection);
     }
 
+    /**
+     * Does nothing since a state of a projection should not be necessarily
+     * updated during execution of a {@linkplain io.spine.core.Subscribe subscriber} method.
```
Comment (armiol) [inline server/src/main/java/io/spine/server/projection/ProjectionEndpoint.java]: `upon execution`

File: server/src/test/proto/spine/test/procman/commands.proto [hunk 3]
```diff
@@ -42,3 +42,7 @@ message PmAddTask {
 message PmStartProject {
     ProjectId project_id = 1;
 }
+
+message PmDoNothing {
```
Comment (armiol) [inline server/src/test/proto/spine/test/procman/commands.proto]: Please have a comment for this particular message. It looks weird.

**Pull request merged**